src/OmniSharp/Workers/Intellisense/SnippetGenerator.cs

```diff
@@ -75,9 +78,9 @@ private void RenderMethodSymbol(IMethodSymbol methodSymbol)
                 }
                 _sb.Append(">");
             }
-            
+
             RenderParameters(methodSymbol);
-            if (methodSymbol.ReturnsVoid)
+            if (methodSymbol.ReturnsVoid && IncludeMarkers)
```

Comment:
No, you're right. Snippet contains the ; not CompletionText. Damn, this is confusing :)